src/viam/components/input/client.py

```diff
@@ -7,6 +7,7 @@
 from grpclib import GRPCError, Status
 from grpclib.client import Channel
 
+
```

Comment:
(minor) should we have this newline? I thought only one linebreak separating imports was standard (disregard if I'm mistaken!).